package.json

```diff
@@ -83,6 +83,21 @@
           "dark": "images/dark/icon-link.svg",
           "light": "images/light/icon-link.svg"
         }
+      },
+      {
+        "command": "java.view.package.revealFileInOS",
```

Comment:
revealFileInOS -> revealInExplorer

Reply:
`revealInExplorer` is not appropriate since it is related to two built-in commands in VS Code: `revealInExplorer` and `git.revealInExplorer`: They all refer to the meaning of "Reveal the file node in VS Code File Explorer", not the OS explorer.

In contrast, `revealFileInOS` is a built-in command of VS Code to open a file in OS explorer, and our command just forwards the dependency node's uri to it, so this naming would be fine.

Reply:
so why not to mount the VS Code command to the dependency explorer directly?

Reply:
It would be best that we could mount the builtin command directly. 
But the builtin command accepts a `Uri` argument, where context entry specified in `package.json` only allows for passing a `ExplorerNode` to the command binding.

(The `TreeItem.command` property can indeed specify the argument to be passed, but this is not the case with `view/item/context` entry)

Reply:
It seems VS Code provides different names in different OSes, could you check whether we need keep consistent?

Reply:
Sorry for I didn't notice this comment...

As [this file](https://github.com/microsoft/vscode/blob/0dbe568b59c9fd213bddc5ddc730a4892f3b011c/src/vs/workbench/contrib/files/electron-browser/fileActions.contribution.ts#L64-L67) shows, VS Code does provide different names here:
* Windows: `Reveal In Explorer`
* Mac OS: `Reveal In Finder`
* Linux: `Open Containing Folder`

And the [Chinese translation](https://github.com/microsoft/vscode-loc/blob/257d355e31d868801fb36be24c6070a3c39885a7/i18n/vscode-language-pack-zh-hans/translations/main.i18n.json#L3905-L3910) uses that localization:
```json
"vs/workbench/contrib/files/electron-browser/fileActions.contribution": {
    "revealInWindows": "在资源管理器中显示",
    "revealInMac": "在 Finder 中显示",
    "openContainer": "打开所在的文件夹",
    "filesCategory": "文件"
},
```

Currently I have no knowledge of how to utilize the localization here according to OS...

Reply:
Currently i'm OK to keep the same one, if there is user complaint, then fix it.
A workaround is add three difference commands, and use when clause to enable the correct one.